spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -419,6 +424,20 @@ describe('GOVUK.analyticsGA4.linkTracker', function () {
         expect(window.dataLayer[0]).toEqual(expected)
       }
     })
+
+    it('detects files with preview in their filename as download links instead of preview links', function () {
```

Comment:
Should this commit `Add extra test to ensure preview links are being detected properly` be merged with the first commit? Unclear why it's separate, can you explain?

Reply:
Yep, they should be one commit - I've never merged a commit before which is why they're separate (but James is going to help me with this)